spec/integration/nodesets/rhel7.yaml

```diff
@@ -0,0 +1,24 @@
+HOSTS:
+  master:
+    roles:
+      - master
+      - agent
+      - database
+      - dashboard
+    platform: el-7-x86_64
+    hypervisor: vcloud
```

Comment:
It would be useful to include a nodeset which can run locally as well using vagrant that can be toggled using the ENV variables.